libclc/x86_64-unknown-linux/libspirv/integer/popcount.cl

```diff
@@ -1,3 +1,13 @@
-#include "helpers.h"
+#include <func.h>
+#include <clcmacro.h>
+#include <spirv/spirv.h>
 
-GEN_UNARY_BUILTIN(popcount)
+_CLC_DEFINE_UNARY_BUILTIN(int, __spirv_ocl_popcount, __builtin_popcount, int)
```

Comment:
What's happening here?

Reply:
The way `popcount` is defined in `generic` doesn't play well with the x86 ABI, so we were (and still are) over-riding it in this file. Previously we had our own macro that did what `_CLC_DEFINE_UNARY_BUILTIN` does (in `libclc/x86_64-unknown-linux/libspirv/integer/helpers.h`), so I've deleted it and decided to go with `_CLC_DEFINE_UNARY_BUILTIN`.